Sources/Player/OfflineEngine/Internal/MediaDownloader.swift

```diff
@@ -60,6 +60,43 @@ private extension MediaDownloader {
 	func createTask(for url: URL) -> URLSessionDownloadTask {
 		urlDownloadSession.downloadTask(with: url)
 	}
+
+	func calculateHLSStreamSize(for path: URL) -> UInt64 {
+		let fileManager = FileManager.default
```

Comment:
[suggestion] I think we can use `FileManagerClient`, we would need to add the `enumerator` there.

Reply:
Done!